tests/regression/tests/REQUEST-933-APPLICATION-ATTACK-PHP/933151.yaml

```
@@ -47,27 +47,27 @@ tests:
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             method: GET
             port: 80
-            uri: /?foo=filemtime%28%24foo%29
+            uri: "/get?foo=filemtime%28%24foo%29"
           output:
             no_log_contains: id "933151"
   - test_title: 933151-4
-    desc: pmf + chain; must run test in PL2!
+    desc: "pmf + chain"
     stages:
       - stage:
           input:
-            data: gethostbynamE(
             dest_addr: 127.0.0.1
             headers:
               Host: localhost
               User-Agent: "OWASP CRS test agent"
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             method: POST
             port: 80
-            uri: /gethost
+            uri: "/post"
+            data: gethostbynamE(
```

```suggestion
            data: "gethostbynamE("
```
Looks like we almost always use quotes around data, do we want to be consistent also here?